pkgs/applications/networking/browsers/vivaldi/default.nix

```diff
@@ -85,6 +74,6 @@ in stdenv.mkDerivation rec {
     homepage    = "https://vivaldi.com";
     license     = licenses.unfree;
     maintainers = with maintainers; [ otwieracz nequissimus ];
-    platforms   = platforms.linux;
+    platforms   = with platforms; intersectLists linux x86_64;
```

Comment:
In fact there are a few. The idea came from searching the repository.

```
$ grep -r 'platforms.*intersectLists.*x86_64'
pkgs/applications/networking/browsers/vivaldi/default.nix:    platforms   = with platforms; intersectLists linux x86_64;
pkgs/applications/misc/robomongo/default.nix:    platforms = stdenv.lib.intersectLists stdenv.lib.platforms.linux stdenv.lib.platforms.x86_64;
pkgs/development/libraries/wiredtiger/default.nix:    platforms = intersectLists platforms.unix platforms.x86_64;
```

But I can use the literal string if you think it is better.